packages/admin/admin/src/form/FinalFormSelect.tsx

```diff
@@ -120,11 +123,12 @@ export const FinalFormSelect = <T,>({
                         {getOptionLabel(value)}
                     </MenuItem>
                 ))}
-            {options.map((option: T) => (
-                <MenuItem value={getOptionValue(option)} key={getOptionValue(option)}>
-                    {getOptionLabel(option)}
-                </MenuItem>
-            ))}
+            {!loading &&
```

Comment:
Should we hide the existing (though stale) options while loading? I'm not sure if this is great UX.

https://github.com/user-attachments/assets/f0c4bf71-ec3e-4352-aba0-206c1340a810

Also, why is the selected option reset while loading?

Reply:
> Should we hide the existing (though stale) options while loading? I'm not sure if this is great UX.

@manuelblum what do you think about this?

Reply:
I would hide the stale options while loading - maybe UX has another opinion on that?

Reply:
Discussed this with UX internally and decided to change the implementation to load the options upon rendering, not opening the dropdown. We will do this in a follow-up PR.